rebar.lock

```diff
@@ -96,7 +96,7 @@
  {<<"leveled">>,
   {git,"https://github.com/martinsumner/leveled.git",
       {ref,"5bc137e4ef90d55f1e0da216a5ef5801d2a3d813"}},
-  0},
+  1},
```

Comment:
I'm sure this digit is not too relevant?! But should we really change it?

Reply:
Idk, it was autogenerated by shipped rebar

Reply:
Sounds like rebar, changing something without any reason 👍